template/README.md

```diff
@@ -1,23 +1,39 @@
 ---
-author: Please fill in the author name here.
-title: Please fill in the title of the feature here.
-description: Please fill in the description of the feature here.
-tags: Please fill in the list of tags here.
-optuna_versions: Please fill in the versions of Optuna in which you have confirmed the feature works.
-license: Please fill in the license name here. 
+author: "Please fill in the author name here."
+title: "Please fill in the title of the feature here."
+description: "Please fill in the description of the feature here."
+tags: ["Please fill in the list of tags here."]
+optuna_versions: ["Please fill in the list of versions of Optuna in which you have confirmed the feature works, e.g., 3.6.1."]
+license: "MIT License"
 ---
 
+<!--
+This is an example of the frontmatters.
+
+---
+author: "Optuna team"
+title: "My Sampler"
+description: "A description for My Sampler."
+tags: ["sampler", "2nd tag for My Sampler", "3rd tag for My Sampler"]
+optuna_versions: ["3.5.0", "3.6.1"]
+license: "MIT License"
+---
+-->
+
 Please read the [tutorial guide](https://optuna.github.io/optunahub-registry/recipes/001_first.html) to register your feature in OptunaHub.
 You can find more detailed explanation of the following contents in the tutorial.
+Looking at [other packages' implementations](https://github.com/optuna/optunahub-registry/tree/main/package) will also help you.
 
 ## Class or Function Names
-Please fill in the class/function name which you implement here. 
+Please fill in the class/function name which you implement here.
 
 ## Installation
 If you have additional dependencies, please fill in the installation guide here.
+If no additional dependencies is required, this section can be removed.
 
 ## Example
 Please fill in the code snippet to use the implemented feature here.
 
 ## Others
-Please fill in any other information if you have here.
+Please fill in any other information if you have here by adding child sections (###).
+If there is no additional information, this section can be removed.
```

Comment:
It would be great to add an example of this section. I think the `Reference` section is an good example.

Reply:
I've added examples.